src/views/ProjectsView.tsx

```diff
@@ -40,6 +40,13 @@ const useStyles = (props: Props) =>
       fontSize: "21px",
       marginRight: "125px",
     },
+    cancelButton: {
+      textTransform: "none",
+    },
+    OKButton: {
+      backgroundColor: theme.palette.primary.main,
```

Comment:
Rather than a class, shouldn't this be a materialUI primary button?

![image](https://user-images.githubusercontent.com/3502798/129691510-58add5b3-9f23-4635-abdd-50d23eddfcde.png)

Reply:
I have made the background blue on hover, similar to the "Enquire" button on the main site https://gliff.ai/